ui/jstests/test-learning-resource.jsx

```diff
@@ -681,8 +682,62 @@ define(['QUnit', 'jquery', 'react', 'lodash', 'learning_resources',
     function(assert) {
       var div = document.createElement("div");
       assert.ok($(div).html().length === 0);
-      LearningResources.loader("repo", "1", function() {}, function() {}, div);
+      LearningResources.loader(
+        "repo",
+        "1",
+        function() {},
+        function() {},
+        false,
+        true,
+        div
+      );
       assert.ok($(div).html().length > 0);
     }
   );
+
+  QUnit.test('Assert learning resource panel does close',
+    function (assert) {
+      var done = assert.async();
+      var closeLearningResourcePanelCount = 0;
+      var closeLearningResourcePanel = function() {
+        closeLearningResourcePanelCount += 1;
+      };
+
+      var afterMountLRPanel = function (component) {
+        waitForAjax(3, function () {
+          if (component.state.closeLRPanel) {
+            assert.equal(closeLearningResourcePanelCount, 1);
+          }
+          done();
+        });
+      };
+
+      var Container = React.createClass({
+        getInitialState: function() {
+          return {closeLRPanel: false};
+        },
+        render: function() {
+          return <LearningResourcePanel
+            repoSlug="repo"
+            closeLearningResourcePanel={this.props.closeLearningResourcePanel}
+            closeLRPanel={this.state.closeLRPanel}
+            learningResourceId="1"
+            refreshFromAPI={function() {}}
+            ref={this.props.afterMountCallback}/>;
+        }
+      });
+
+      var afterMount = function (component) {
+        component.setState({closeLRPanel: true}, function () {
+          assert.equal(component.state.closeLRPanel, true);
+        });
+      };
```

Comment:
You are missing a `done` here. You can make two `done` functions by calling `assert.async()` twice.